test/batch.js

```diff
@@ -23,17 +24,24 @@ test('create nodes', function (t) {
     }
   ]
 
+  var updatedWithLinks = utils.deepClone(nodes)
+  updatedWithLinks[0].links = []
+  updatedWithLinks[1].links = []
+
   var batch = nodes.map(function (node) {
     return {
       type: 'put',
       value: node
     }
   })
+  var batchClone = JSON.parse(JSON.stringify(batch))
```

Comment:
Consider using `clone` here too, just so it's clear to future readers what your intention was.